pkg/vmprovider/providers/vsphere/virtualmachine/powerstate.go

```diff
@@ -32,6 +37,10 @@ func ChangePowerState(
 		return errors.Wrapf(err, "failed task creation to change power state to %s", ps)
```

Comment:
Nit: if `ShutdownGuest()` fails this error is now a little misleading. Mostly implementation detail so ignore comment if you don't feel like shuffle code around or whatnot.

Reply:
Resolved by returning a different message for ShutdownGuest.